composer.json

```diff
@@ -27,14 +27,15 @@
     "require-dev": {
         "filp/whoops": "^1.1 || ^2.0",
         "malukenho/docheader": "^0.1.5",
+        "mikey179/vfsStream": "^1.6",
         "mockery/mockery": "^0.9.5",
         "phpunit/phpunit": "^5.0",
+        "zendframework/zend-code": "^3.1",
```

Comment:
New script "expressive-scan-for-error-middleware" requires `zend-code` to work, but this is only in `require-dev` section. So to use the script in project it's required to install `zend-code`. I think it should be noted in documentation, because this tool is not gonna work out of the box.